### PR TITLE
Fixed #25941 -- Improved error message for runtests.py when django is not on path.

### PR DESCRIPTION
**Pull request by ThomasAllison**
Catches the ImportError in the runtest.py and outputs an
improved message. The message directs the user to the documentation.
The steps in the tests/README.rst solve the ImportError.

Ticket: https://code.djangoproject.com/ticket/25941

### CODE REVIEW

File: tests/runtests.py [hunk 1]
```diff
@@ -10,7 +10,12 @@
 import tempfile
 import warnings
 
-import django
+try:
+    import django
+except ImportError:
+    sys.stdout.write("Django module not found, please reference tests/README.rst for instructions.\n")
```
Comment (apollo13) [inline tests/runtests.py]: is there a reason to use `sys.stdout.write` instead of just print? also error messages should probably go to `stderr`

Comment (ThomasAllison) [reply tests/runtests.py]: No, and I agree `stderr` would be better. Will change that.

Comment (apollo13) [reply tests/runtests.py]: I think `print(… file=sys.stderr)` would be better.

Also what about something like:
```
 try:
    import django
except ImportError as e:
   raise RuntimeError("Django module not found, please reference tests/README.rst for instructions.") from e
```

which would keep the traceback?

File: tests/runtests.py [hunk 2]
```diff
@@ -10,7 +10,12 @@
 import tempfile
 import warnings
 
-import django
+try:
+    import django
+except ImportError:
+    sys.stderr.write("Django module not found, please reference tests/README.rst for instructions.\n")
```
Comment (felixxm) [inline tests/runtests.py]: Please use single quotes.

File: tests/runtests.py (hunk 1 above)
Comment (ThomasAllison) [reply tests/runtests.py]: That would probably be the desired behavior.

**Pull request merged**